lasso_munging.R

```diff
@@ -0,0 +1,61 @@
+source("./load_data_by_sex.R")
+library(regmed)
+library(dplyr)
+library(haven)
+
+male_data()
+female_data()
+
+munge_lasso <- function(sex) {
+  if (sex == "male"){
+    d = df_male_domsp_weekday
```

Comment:
You could just pass the dfs into the function to avoid needing this if statement